src/tooltip/tooltip-container.component.ts

```diff
@@ -42,6 +42,8 @@ export class TooltipContainerComponent implements AfterViewInit {
   containerClass: string;
   animation: boolean;
 
+  @HostBinding('attr.id') ariaDescribedby: string;
```

Comment:
attribute `id` is bound to `ariaDescribedby`? doesn't seem very logic to me. change property name to `id` or `containerId`

Reply:
fixed